src/main/java/seedu/address/logic/commands/AddTodoCommand.java

```diff
@@ -0,0 +1,74 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.project.Project;
+import seedu.address.model.task.CompletableTodo;
+import seedu.address.model.task.todo.Todo;
+
+public class AddTodoCommand extends Command {
+
+    public static final String COMMAND_WORD = "addTto";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a Task to CoLAB. "
+            + "Parameters: "
+            + PREFIX_DESCRIPTION + "DESCRIPTION "
```

Comment:
You forgot to include the project-index in this message_usage

Reply:
Good catch. Added it. Thanks!